paragon/_variables.scss

```diff
@@ -1,1311 +1,1311 @@
-$element-color-levels: () !default;
```

Comment:
Temporarily comments out all the SCSS variables as this PR intends to generate variables with `style-dictionary` instead.